libs/blocks/global-footer/global-footer.js

```diff
@@ -226,6 +226,12 @@ class Footer {
     if (url.hash !== '') {
       // Hash -> region selector opens a modal
       decorateAutoBlock(regionPickerElem); // add modal-specific attributes
+      if (regionPickerElem.classList[0] !== 'modal') {
```

Comment:
A TODO to remove those logs again, might make sense after MWPW-000000 is completed. (It will 100% be forgotten and stay here forever, but at least there might be a chance we remove this again 😁 )

Reply:
Good idea 👍 I added a TODO